pkg/harness/stackrun/v1/types.go

```diff
@@ -17,6 +17,9 @@ type StackRun struct {
 	ExecWorkDir *string
 	Approval    bool
 	ApprovedAt  *string
+	ManageState bool
+	User       *gqlclient.UserFragment
```

Comment:
there's a different field you should sideload for plural provider auth, `pluralCreds`.  The actual actor user is likely not helpful (beyond ultimately being the user those creds are generated from).

It's also expected this will be nil pretty often in which case set no env vars